cli/turbo.json

```diff
@@ -3,8 +3,18 @@
   "extends": ["//"],
   "pipeline": {
     "build": {
-      "env": ["GO_TAG"],
-      "outputs": ["turbo", "turbo.exe"]
+      "outputs": [
+        "../target/debug/go-turbo",
+        "../target/debug/turbo",
+        "../target/release/go-turbo",
+        "../target/release/turbo"
+      ],
+
+      "inputs": [
+        "**/*.go",
+        "**/*_test.go",
+        "../crates/turborepo-*/**" // Rust crates
```

Comment:
@NicholasLYang going to block this on #4853, so it's easier to specify the dependencies. Let me know if there's a better way to do this too